### DISCUSSION TITLE
Tic Tac Toe

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#83] [Games]
# Tic Tac Toe

Build retro games using WebAssembly for a fantasy console

https://wasm4.org/play/tictactoe

**Comment by christopher-kleine** [comment]
This is a simple version of Tic Tac Toe written in Go.
Arrow Keys = D-Pad
X = Place your own piece.
Y/Z/C = Tell the A.I. to place a piece.

[Source Code](https://github.com/christopher-kleine/tic-tac-toe-wasm4)
